scripts/fuzz_loop.sh

```diff
@@ -9,7 +9,7 @@ FUZZ_COMMAND=$1
 
 while true; do
```

Comment:
I wonder if anyone uses these scripts...

Reply:
The easiest way to find out would be to kill them :D